scripts/github_api

```diff
@@ -76,6 +76,12 @@ class Comment:
     https://developer.github.com/v3/issues/comments/
     """
     def __init__(self, repo, issue_number):
+        """
+        Args:
+            repo (str): target repository path. Format
+                <repo owner>/<repo>
+            issue_number (str): number of the issue or pull-request
+        """
```

Comment:
This file's modifications belong to the previous commit, which creates those functions.